h2o-py/h2o/estimators/stackedensemble.py

```diff
@@ -208,3 +210,31 @@ def keep_levelone_frame(self, keep_levelone_frame):
         self._parms["keep_levelone_frame"] = keep_levelone_frame
 
 
+    @property
+    def metalearner_params(self):
+        """
+        Parameters for metalearner algo
```

Comment:
Could we add a short example in the docstring?  e.g. `metalearner_gbm_params = {'max_depth': 2, 'col_sample_rate': 0.3}`

Reply:
Yes

Reply:
Example is added here 93a577c